api/src/main/java/jakarta/enterprise/inject/Instance.java

```diff
@@ -71,11 +71,6 @@
  * Instance&lt;PaymentProcessor&gt; anyPaymentProcessor;
  * </pre>
  * 
- * <p>
- * Finally, the {@link New &#064;New} qualifier may be used, allowing the application to obtain a
- * {@link New &#064;New} qualified bean:
- * </p>
- * 
```

Comment:
Probably want to remove the next paragraph as well?

Reply:
I mean, not a paragraph, but a `<pre>` block.

Reply:
Yep thanks, missed that.
And there is similar case in `injectionandresolution` file - will address both!

Reply:
Should be good now :)